Add long-press action to copy subject text to clipboard

jnferreira opened the pull request:
Fixes issue [#4919](https://github.com/k9mail/k-9/issues/4919). 
It was implemented a long click on the email subject area, which copies it to the clipboard.

cketti reviewed (changes requested):
Thank you for the pull request :heart: 

The mechanism (long-press) is consistent with what we currently use for other fields. So that's great :+1:

The pull requests needs some additional work, though. 

1. You included some images, presumably because you're doing this for a university course and need some kind of documentation for your work. By updating the branch you created this pull request from you also update the pull request. But we don't want these images in our repository. I suggest creating a different branch for your documentation and resetting this branch to only contain the commit containing the feature.
2. The commit message is a bit confusing. I suggest changing it to something like: "Add long-press action to copy subject text to clipboard".
3. The implementation needs some changes. I added individual comments for this.

I deliberately left out some details on how to accomplish certain tasks. The idea is that you'll have to figure things out on your own, hopefully learning new things in the process. However, if you're getting stuck, please don't hesitate to ask for help.

jnferreira commented:
> 
> 
> Thank you for the pull request ❤️
> 
> The mechanism (long-press) is consistent with what we currently use for other fields. So that's great 👍
> 
> The pull requests needs some additional work, though.
> 
>     1. You included some images, presumably because you're doing this for a university course and need some kind of documentation for your work. By updating the branch you created this pull request from you also update the pull request. But we don't want these images in our repository. I suggest creating a different branch for your documentation and resetting this branch to only contain the commit containing the feature.
> 
>     2. The commit message is a bit confusing. I suggest changing it to something like: "Add long-press action to copy subject text to clipboard".
> 
>     3. The implementation needs some changes. I added individual comments for this.
> 
> 
> I deliberately left out some details on how to accomplish certain tasks. The idea is that you'll have to figure things out on your own, hopefully learning new things in the process. However, if you're getting stuck, please don't hesitate to ask for help.

Thank you very much for your quick response!

We have corrected the details that you mentioned and I hope we have corrected them. Thanks again for your quick response and all the explanation about the fixes. We hope to have contributed well to the project :)

cketti commented:
Thanks :+1:
